src/v1/steward.ts

```diff
@@ -509,6 +524,10 @@ export function handleForeclosure(event: Foreclosure): void {
   let patronString = foreclosedPatron.toHexString();
   let foreclosedTokens: Array<string> = [];
 
+  // log.info('[[[[[[[[[[FROM INSIDE handleForeclosure]]]]]]]]]]]]]]]: {}, {}, {}', [
+  //   txHashString.toString(),
+  // ])
```

Comment:
Don't fortget to clean this up when you are happy.

Reply:
for sure, shouldn't have left it in